src/adapter_wrap.cc

```diff
@@ -44,6 +54,23 @@ void AdapterWrap::OnUvAsync(uv_async_t *req, int status) {
     EmitEvent(asyncData->argThis, argc, argv);
     asyncData->logQueue.pop();
   }
+  while(!asyncData->commandQueue.empty()) {
+        CEC::cec_command command = asyncData->commandQueue.front();
+        
+        const unsigned argc = 2;
+        Local<Object> data = Object::New();
+        data->Set(String::NewSymbol("initiator"), Number::New(command.initiator));
+        data->Set(String::NewSymbol("destination"), Number::New(command.destination));
+        data->Set(String::NewSymbol("opcode"), Number::New(command.opcode));
+        data->Set(String::NewSymbol("parameter000"), Number::New(command.parameters.data[0]));
```

Comment:
hmm, the cectypes.h header specifies that the parameters property is of type cec_datapacket, which itself has up to 100 bytes. wouldn't it make more sense to convert this datapacket to a Buffer type and only have a single "parameter" key?